src/index.js

```diff
@@ -1 +1,68 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+
+// Send Data to API ==> Create a new Game
+
+const api = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+
+fetch(api, {
+  method: 'POST',
+  body: JSON.stringify({
+    name: 'Maze Runner',
+  }),
+  headers: {
+    'Content-type': 'application/json; charset=UTF-8',
+  },
+});
+
+// Send Data to API ==> Create a new score for the given game
+
+const form = document.querySelector('#add_score_info');
+const userName = document.querySelector('.user_name');
+const score = document.querySelector('.user_score');
+
+const newScoreAndUser = async () => {
+  await fetch(`${api}nSxqOE1SzPIiMx7pzKks/scores/`, {
+    method: 'POST',
+    body: JSON.stringify({
+      user: userName.value,
+      score: score.value,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+};
```

Comment:
- [ ] The name should be a text not a number, kindly make sure that a user doesn't send the number instead of text.

![db](https://user-images.githubusercontent.com/55635977/150992634-b790eb15-c3cb-49d1-a2b2-a6015b5f8cc0.png)